tests/post/steps/test_authentication.py

```diff
@@ -81,6 +81,50 @@ def control_plane_ip(host):
 
 # Given {{{
 
+
+@given(parsers.parse(
+    "the control-plane Ingress container is Ready"))
+def check_cp_ingress_containers(request, host, k8s_client):
```

Comment:
Can't you use the same `given` you use for `And pods with label 'app.kubernetes.io/name=dex' are 'Ready'` ? To me, it looks like the same.

Reply:
The extra check for container status has been added to the same function that checks ingress pods.
so now the function has the following submethods
- check ingress pod status
- obtain the full name of ingress pods
- then finally check the status of containers in that pod